def exception_handler(CustomBaseException):
    return None